packages/zwave-js/src/lib/commandclass/MultilevelSwitchCC.ts

```diff
@@ -109,6 +111,33 @@ function getSuperviseStartStopLevelChangeValueId(): ValueID {
 	};
 }
 
+export function getCompatEventValueId(endpoint?: number): ValueID {
+	return {
+		commandClass: CommandClasses["Multilevel Switch"],
+		endpoint,
+		property: "event",
+	};
+}
+
+/**
+ * @publicAPI
+ * This is emitted when a start or stop event is received
+ */
+export interface ZWaveNotificationCallbackArgs_MultilevelSwitchCC {
+	eventType: MultilevelSwitchCommand;
```

Comment:
```suggestion
	eventType: MultilevelSwitchCommand.StartLevelChange | MultilevelSwitchCommand.StopLevelChange;
```
would be more specific